src/xGadgetron/cGadgetron/gadgetron_data_containers.cpp

```diff
@@ -59,6 +60,51 @@ MRAcquisitionData::write(const char* filename)
 	}
 }
 
+void
+MRAcquisitionData::read( const std::string& filename_ismrmrd_with_ext )
+{
+	
+	bool const verbose = true;
+
+	if( verbose )
+		std::cout<< "Started reading acquisitions from " << filename_ismrmrd_with_ext << std::endl;
+	try
+	{
+
+		ISMRMRD::Dataset d(filename_ismrmrd_with_ext.c_str(),"dataset", false);
+
+		std::string xml;
+		d.readHeader(xml);
+		this->acqs_info_ = xml;
```

Comment:
could these three lines be replaced with `d.readHeader(this->acqs_info_);`?

Reply:
I don't think so.
I intended the reader to be employed in this way:

```
std::string filename_h5 = "dummy.h5";
sirf::AcquisitionsVector some_vector;
some_vector.read( filename_h5 ); //fills the data and sets the acquisitions_info
```
and the acquisitions info will be set matching the file which was read in. Default-constructed `AcquisitionsVector` have empty `acqs_info_`  members.

Reply:
Ok, you know best!

Reply:
ah darn you are right... ( which means you were wrong with "Ok, you know best!" though xD ). I'll fix it.